api/v1beta1/vmnodescrape_types.go

```diff
@@ -0,0 +1,112 @@
+package v1beta1
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// VMNodeScrapeSpec defines discovery for targets placed on kubernetes nodes,
+// usually its node-exporters and other host services.
+// InternalIP is used as __address__ for scraping.
+type VMNodeScrapeSpec struct {
```

Comment:
Just a question: Why did you design `VMNodeScrapeSpec` as a "flat" struct? `VMServiceScrapeSpec` has `selector` and `sampleLimit` etc as its direct children while `port` and `relabelConfigs` etc are inside `endpoints` slice (array). Are there any special reason?

Reply:
@umezawatakeshi It's because I requested so https://github.com/VictoriaMetrics/operator/issues/118#issuecomment-739917475

Reply:
@ymmt2005 Oh, understood.